awx/plugins/inventory/foreman.ini.example

```diff
@@ -152,6 +152,12 @@ want_hostcollections = False
 # Disabled by default as the change would else not be backward compatible.
 rich_params = False
 
+# Whether to populate the ansible_ssh_host variable to explicitly specify the
+# connection target.  Only tested with Katello (Red Hat Satellite).
+# If the foreman 'ip' fact exists then the ansible_ssh_host varibale is populated
+# to permit connections where DNS resolution fails.
+want_ansible_ssh_host = False
```

Comment:
Adding this to the example doesn't actually do anything.

To allow users to _use_ this, wouldn't we need to pass-through data from inventory source `source_vars` to the INI file we write for this source?

https://github.com/ansible/awx/blob/42562e86e4769032a2dab9f9eda4ed01e962acd3/awx/main/tasks.py#L1930-L1935

Granted, the only reason I'm able to make this suggestion is because developing new testing for the laid-down ini files for inventory imports was literally the last thing I was literally the last thing I was doing before I opened your PR.